modular_pariah/modules/radiosound/readme.md

```diff
@@ -0,0 +1,29 @@
+ttps://github.com/Skyrat-SS13/Skyrat-tg/pull
```

Comment:
```suggestion
https://github.com/Skyrat-SS13/Skyrat-tg/pull
```
This is also missing the PR ID

Reply:
Seems to be from the [original PR](https://github.com/Skyrat-SS13/Skyrat-tg/blob/20f1aded5d8d8bec15be67dfd3db963ec248884b/modular_skyrat/modules/radiosound/readme.md?plain=1#L1) actually, not sure how that was missed in reviews since it's right at the top of the file.

Reply:
I had heard offhand comments about how SR's dev team missed a lot of super obvious stuff, but didn't think much of it.

Seeing this has made me think otherwise.